geode-for-redis/src/distributedTest/java/org/apache/geode/redis/internal/commands/executor/list/LPopDUnitTest.java

```diff
@@ -144,18 +144,19 @@ private void lpopPerformAndVerify(String key, AtomicLong runningCount) {
 
     int elementCount = INITIAL_LIST_SIZE - 1;
     while (jedis.llen(key) > 0 && runningCount.get() > 0) {
-      String expected = "-" + key + "-" + (elementCount) + "-";
+      String expected = makeElementString(key, elementCount);
       String element = expected;
```

Comment:
`expected` is redundant here, and since the first thing we do inside the try/catch is overwrite the value of `element`, this can become:
```
      String element = "";
```